tests/wrappers/test_python_function.py

```diff
@@ -26,7 +26,7 @@ def setup_function(func):
 def test_function_wrapper_sanity(_, ):
     retval = 'success'
 
-    @epsagon.wrappers.python_function.python_wrapper
+    @epsagon.wrappers.python_function.python_wrapper(name='test-func')
```

Comment:
I think it is better to add a test (duplicate this one, share relevant code) for python_wrapper with name. That way we still have a test for without a name and see that it didn't break

Reply:
We have other tests that don't use a name. Do you think another one is still needed?